.github/workflows/build.yml

```diff
@@ -3,13 +3,9 @@ on:
     branches:
       - master
       - sq-10
-    paths-ignore:
```

Comment:
the underlying github action was removed so we don't need the path ignore any longer.